docs/generators/ada-server.md

```diff
@@ -6,6 +6,7 @@ sidebar_label: ada-server
 | Option | Description | Values | Default |
 | ------ | ----------- | ------ | ------- |
 |allowUnicodeIdentifiers|boolean, toggles whether unicode identifiers are allowed in names or not, default is false| |false|
+|discriminatorExplicitMappingVerbose|Every model that has a discriminator explicitly defines the disciminator in itself, the mapping in that discriminator includes any descendent schemas that allOf inherit from self, any oneOf schemas, any anyOf schemas, any x-discriminator-values, and the spec's defined discriminator mapping schemas. When true, we validate that oneOf and anyOf schemas contain the required discriminator and we throw an error if it's missing. When true model templates should not loop over {{#children}} when defining the discriminator mapping| |false|
```

Comment:
test

Reply:
Yes, I think that eventually all v3 generators will want to turn on this flag by default. It makes discriminator handling easier and it adds validation checking of the discriminator which is a significant feature addition and will help users.

This description has been updated per your feedback. We now state what tag to iterate over and what tags not to iterate over. Yup the children name is unclear, it is used for class inheritance so we can loop over the child classes and grab their names. We don't have that information in our composed schemas because they are children that allOf include the self schema/class.